tests/parsing/GlobbingTest.py

```diff
@@ -100,6 +101,41 @@ def test_alternatives(self):
                              sorted(alternatives))
 
 
+class EscapingTest(unittest.TestCase):
+
+    def _test_cases(self, in_arr, out_arr):
+        for case in list(zip(in_arr, out_arr)):
+            self.assertEqual(glob_escape(case[0]),
+                             case[1])
+
+    def test_escaping(self):
+        input_strings = [
+            "test",
+            "test[",
+            "test []",
+            "test [[]",
+            "test ]] str [",
+            "test[][]",
+            "test(",
+            "test)",
+            "test()",
+            "test (1)"
+        ]
```

Comment:
--> `"test (1)"]` <-- place the closing-bracket directly behind the last item :)